geode-assembly/src/test/java/org/apache/geode/test/dunit/rules/gfsh/GfshRule.java

```diff
@@ -102,35 +107,52 @@ protected ProcessBuilder toProcessBuilder(GfshScript gfshScript, Path gfshPath,
       commandsToExecute.add("-e " + command);
     }
 
-    return new ProcessBuilder(commandsToExecute).directory(workingDir);
+    ProcessBuilder p = new ProcessBuilder(commandsToExecute);
+    p.directory(workingDir);
+
+    // TODO PSRhomberg -- Input requested: Is this OS agnostic
+    List<String> extendedClasspath = gfshScript.getExtendedClasspath();
+    if (!extendedClasspath.isEmpty()) {
+      Map<String, String> m = p.environment();
+      String classpathKey = "CLASSPATH";
+      String existingJavaArgs = m.get(classpathKey);
+      String specified = String.join(":", extendedClasspath);
```

Comment:
Linux and Mac use ":" as path separator. Use this instead:

org.apache.commons.lang.SystemUtils.PATH_SEPARATOR